test-defaults.ini

```diff
@@ -38,9 +38,7 @@ badge_promo_codes_enabled = True
 
 shiftless_depts = "con_ops",
 
-# we want 2 preassigned types to test some of our logic, so we've
-# set these even though Supporter isn't really a badge type anymore
-preassigned_badge_types = "staff_badge", "supporter_badge"
+preassigned_badge_types = "staff_badge", "contractor_badge"
```

Comment:
This is just for the test suite. Since the original comment said we needed two badge types to test with, I figured I'd replace the Supporter badge I was taking out.

I'm not sure how we'll handle contractor badges, but I'm assuming for now they won't be preprinted.